podcasts/Whats New/WhatsNew.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+struct WhatsNewAnnouncement {
+    let version: Double
+    let image: String
+    let title: String
+    let message: String
+}
+
+class WhatsNew {
+    let announcements: [WhatsNewAnnouncement]
+    let currentVersion: Double
+    let previousOpenedVersion: Double?
+
+    init(announcements: [WhatsNewAnnouncement] = Announcements().announcements, previousOpenedVersion: Double? = UserDefaults.standard.string(forKey: Constants.UserDefaults.lastRunVersion)?.toDouble(), currentVersion: Double = Settings.appVersion().toDouble()) {
+        self.announcements = announcements
+        self.previousOpenedVersion = previousOpenedVersion
+        self.currentVersion = currentVersion
+    }
+
+    func viewControllerToShow() -> UIViewController? {
+        guard let previousOpenedVersion,
+              let announcement = announcements.filter { $0.version >= previousOpenedVersion && $0.version <= currentVersion }.last else {
```

Comment:
Based on this logic a user will always see the announcement if it targets the currently running version, even after they have updated (second launch on the new version)

```suggestion
              let announcement = announcements.filter { $0.version > previousOpenedVersion && $0.version <= currentVersion }.last else {
```